docs/Changelog.md

```diff
@@ -2,6 +2,7 @@
 
 1.13.0 (Not released yet)
 -------------------------
+* [#4988](https://github.com/TouK/nussknacker/pull/4988) Method definition `def authenticationMethod(): Auth[AuthCredentials, _]` was changed to `def authenticationMethod(): EndpointInput[AuthCredentials]`
```

Comment:
are we able to edit it and write it from the user perspective (the user is a Nu extension developer) without mentioning the implementation details (we have them in the migration guide)?

Reply:
You are right - I think that we can use `Refactor: Allow to use custom authentication methods`. I've just committed change - It is generic but it states what can be done. WDYT?

Reply:
S(ounds)GTM :slightly_smiling_face: